website/content/en/docs/cloud-providers/AWS.md

```diff
@@ -0,0 +1,193 @@
+instance type to maximize node utilization.
+
+The default value includes all instance types with the exclusion of metal
+(non-virtualized),
+[non-HVM](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/virtualization_types.html),
+and GPU instances. View the full list with `aws ec2 describe-instance-types`.
+
+**Example**
+
+*Set Default with provisioner.yaml*
+
+```yaml
+spec:
+  instanceTypes:
+    - m5.large
+```
+
+*Override with pod.yaml*
+
+```yaml
+spec:
+  template:
+    spec:
+      nodeSelector:
+        node.k8s.aws/instance-type: m5.large
+```
+
+## Node Lables in Provisioner Spec
+
+### Launch Template
+
+Karpenter uses [AWS Bottlerocket](https://aws.amazon.com/bottlerocket/) by default. 
+
+You can specify a different [launch
+template](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-launch-templates.html),
+such as a customized version of Bottlerocket or a different platform
+altogether.
+
+**Template ID**
+- key: `node.k8s.aws/launch-template-id`
+- value example: `bottlerocket` (default)
+- value list: `aws ec2 describe-launch-templates`
+
+**Template Version**
+- key: `node.k8s.aws/launch-template-version`
+- value example: `3`
+- default: `$LATEST`
+- value list: `aws ec2 describe-launch-templates`
+
+### Capacity Type (e.g., spot)
+
+- key: `node.k8s.aws/capacity-type`
+- values
+  - `on-demand` (default)
+  - `spot`
+
+Karpenter supports specifying capacity type and defaults to on-demand.
+
+Specify this value on the provisioner to enable spot instance pricing. [Spot
+instances](https://aws.amazon.com/ec2/spot/) may be preempted, and should not
+be used for critical workloads.
+
+**Example**
+
+*Set Default with provisioner.yaml*
+
+```yaml
+spec:
+  labels: 
+    node.k8s.aws/capacity-type: spot
+```
+
+*Override with pod.yaml*
+
+```yaml
+spec:
+  template:
+    spec:
+      nodeSelector:
+        node.k8s.aws/capacity-type: spot
+```
+
+### Architecture (e.g., ARM) 
+
+- key: `kubernetes.io/arch`
+- values
+  - `amd64` (default)
+  - `arm64`
+
+Karpenter supports `amd64` (e.g., intel) nodes, and `arm64` nodes. The default
+is `amd64`.
+
+**Example**
+
+*Set Default with provisioner.yaml*
+
+```yaml
+spec:
+  labels: 
+    kubernetes.io/arch: arm64
+```
+
+*Override with pod.yaml*
+
+```yaml
+spec:
+  template:
+    spec:
+      nodeSelector:
+        kubernetes.io/arch: amd64
+```
+
+## Operating System
+
+- key: `kubernetes.io/os`
+- values
+  - `linux` (default)
+
+At this time, Karpenter only supports linux nodes.
+
+### AWS Region
+
+`topology.kubernetes.io/region=us-east-1`
+
+- key: `topology.kubernetes.io/region`
+- value example: `us-east-1`
+- value list: [AWS regional endpoints](https://docs.aws.amazon.com/general/latest/gr/rande.html)
+
+Karpenter can be configured to create nodes in a particular region.
+
+### Availability Zones
+
+`topology.kubernetes.io/zone=us-east-1c`
+
+- key: `topology.kubernetes.io/zone`
+- value example: `us-east-1c`

```

Comment:
We also support Zone IDs on this field in addition to zone names, since zone names are not necessarily aligned across accounts. https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html

Reply:
oh i didn't know you could actually resolve that between accounts, good to know 👍